requirements/edx/testing.txt

```diff
@@ -624,7 +624,7 @@ execnet==1.9.0
     # via pytest-xdist
 factory-boy==3.2.1
     # via -r requirements/edx/testing.in
-faker==11.3.0
+faker==12.0.0
```

Comment:
Only an update in dependencies. No major [change](https://github.com/joke2k/faker/blob/master/CHANGELOG.md#v1200---2022-01-31
) in new version.